android/sdl_android/src/main/java/com/smartdevicelink/haptic/HapticInterfaceManager.java

```diff
@@ -91,6 +93,15 @@ private void findHapticRects(View root, final List<HapticRect> hapticRects) {
         List<View> focusables = new ArrayList<>();
         getFocusableViews(root, focusables);
 
+        double scale = 1.0;
+        ISdl proxy = proxyHolder.get();
```

Comment:
Potential NPE. This needs to be wrapped in a `if (proxyHolder.get() != null){ `

Reply:
The condition is there but in the caller method: refreshHapticData
https://github.com/smartdevicelink/sdl_java_suite/blob/30b3ba4e6e93b8b66133b3eb971887d5286b0440/android/sdl_android/src/main/java/com/smartdevicelink/haptic/HapticInterfaceManager.java#L77-L88

Reply:
Calling `.get()` on a WeakReference can cause an NPE as there is no guarantee that that reference will be kept. So in the case pointed out, you will need one around line 78 and one around line 97. 

I see that that existed previously, but it is a potential NPE, so it should be fixed. 

https://developer.android.com/reference/java/lang/ref/package-summary.html#reachability

Reply:
DONE